src/components/Tooltip.tsx

```diff
@@ -10,27 +10,30 @@ import { Css } from "src/Css";
 
 interface TooltipProps {
   /** The content that shows up when hovered */
-  title: string;
+  title: ReactNode;
   children: ReactElement;
   placement?: Placement;
   delay?: number;
   disabled?: boolean;
 }
 
 export function Tooltip(props: TooltipProps) {
-  const state = useTooltipTriggerState({ delay: 500, ...props });
+  const { placement, children, title, disabled, delay } = props;
+
+  const state = useTooltipTriggerState({ delay, isDisabled: disabled });
   const triggerRef = React.useRef(null);
-  const { placement, children, title, disabled } = props;
   const { triggerProps, tooltipProps: _tooltipProps } = useTooltipTrigger(
-    { ...props, isDisabled: disabled },
+    { delay, isDisabled: disabled },
     state,
     triggerRef,
   );
   const { tooltipProps } = useTooltip(_tooltipProps, state);
 
   return (
     <>
-      {React.cloneElement(children, { ref: triggerRef, ...triggerProps })}
+      <span ref={triggerRef} {...triggerProps}>
```

Comment:
**Question:** Any advantage to adding a wrapping `span`?

Reply:
I'm pretty sure `cloneElement` ignores the `ref` prop (both `ref` and `key` are ignored, and the children's original ref/key are still used). I'm not really sure if this fixes anything per se, I'd just noticed that "`cloneElement` doesn't support ref" while doing other things.

Reply:
It seems that cloneElement supports `ref`s otherwise the Tooltip wouldn't work (maybe its ignored if it is given) 😢 Are you facing a case that you need to pass a ref to the Tooltip component (that might be why, since it's being overwritten)?

Here's an old conversation on the subject https://github.com/facebook/react/issues/8873#issuecomment-275423780

@oagboola and I approached it this way to reduce the wrapping elements needed for the Tooltip and add the Popper ref to the child element (assuming that the child would not have a ref of its own). All good either way!

Reply:
I was just looking at:

https://reactjs.org/docs/react-api.html#cloneelement

"Clone and return a new React element using element as the starting point. The resulting element will have the original element’s props with the new props merged in shallowly. New children will replace existing children. key and ref from the original element will be preserved."

Reply:
Yeah, it's that preserved behaviour I think is what is special - like if theres are no overrides then it will keep it the original ref. I'll CodeSandbox this for fun!

Reply:
Here's a CodeSandbox showing that the children's ref is overridden by the clone elements ref! Still a very interesting result! 

https://codesandbox.io/s/reactcloneelement-ref-override-gyyhm

Reply:
Oh I see, it "preserves it" as in "it won't drop the existing one", but you're allowed to override it...playing devils advocate, if the caller did want to use the ref, i.e. the `parentRef` in your example, this approach drops it?

I'll put back though, given it's not broken as I originally though.